### DISCUSSION TITLE
HTTP/2

### DISCUSSION THREAD

**Discussion by timfemey** [kartikk221/hyper-express#84] [Q&A] [answered]
Is there a way for me to use HTTP/2 with hyper-express?

**Comment by kartikk221** [comment, marked as answer]
Hello @timfemey It seems that HTTP/2 will not be supported with `uWebsockets.js` which is the low level webserver that powers `hyper-express`. See https://github.com/uNetworking/uWebSockets.js/discussions/178 for more information.

However, you may use `hyper-express` behind a reverse proxy such as NGINX which supports HTTP/2 to be able to provide the client with the ability to retrieve multiple resources in parallel.

It seems that HTTP/3 will be supported in the future as stated in this experimental binary release for `uWebsockets.js`: https://github.com/uNetworking/uWebSockets.js/issues/742

**Reply by timfemey** [reply]
Thanks for the reply
